src/components/leaderboard/leaderboard-tabs/LeaderboardTabs.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Typography, Tabs, Pagination } from 'antd';
+import LeaderboardTab, {
+  LeaderboardItem,
+} from '../leaderboard-tab/LeaderboardTab';
+
+export type TabInfo = {
+  name: string;
+  content: LeaderboardItem[];
+};
+
+type LeaderboardTabsProps = {
+  tabs: TabInfo[];
+  defaultTab?: string;
+  itemsPerPage: number;
+};
+
+const LeaderboardTabs: React.FC<LeaderboardTabsProps> = ({
+  tabs,
+  defaultTab,
+  itemsPerPage,
+}) => {
+  const [currentPage, setCurrentPage] = React.useState<number>(1);
+  const [currentTab, setCurrentTab] = React.useState<string>(tabs[0].name);
+
+  const onChangePage = (page: number): void => setCurrentPage(page);
+  const onChangeTab = (tab: string): void => setCurrentTab(tab);
+
+  return (
+    <Tabs
+      type="card"
+      defaultActiveKey={currentTab}
+      onChange={onChangeTab}
+      tabBarExtraContent={
+        <Pagination
```

Comment:
The colors don't quite work here
https://user-images.githubusercontent.com/15748593/104765558-edc96900-571d-11eb-9bc8-6f001916d3ad.mov